roles/app/templates/env.j2

```diff
@@ -14,6 +14,7 @@ CURRENCY="{{ currency }}"
 CHECKOUT_ZONE="{{ checkout_zone }}"
 
 GOOGLE_MAPS_API_KEY="{{ google_maps_api_key }}"
+GOOGLE_MAPS_REGION="{{ google_maps_region }}"
```

Comment:
Will this be provided in the secrets file? 
If it's not a secret, you could consider including it in `inventory/group_vars/in.yml`

Reply:
Actually, this is a new variable which has not been set for other countries, so it would create an error. Can you please update it to include a condition (like `
{% if s3_access_key is defined %}` below)?

Reply:
Unfortunately, the openfoodnetwork code doesn't guard against an empty string either. Otherwise that would have been valid to define an empty string here.

Reply:
`GOOGLE_MAPS_REGION` is used [on the same line](https://github.com/openfoodfoundation/openfoodnetwork/blob/master/app/views/shared/_google_maps_js.html.haml) as `GOOGLE_MAPS_API_KEY` so I was thinking that we should put close to each other.

Why do you think it will cause an error ?
If it is blank it will not add region parameter.
```
ENV['GOOGLE_MAPS_REGION'] ? '&region=' + ENV['GOOGLE_MAPS_REGION'] : ''
```

Reply:
> `GOOGLE_MAPS_REGION` is used [on the same line](https://github.com/openfoodfoundation/openfoodnetwork/blob/master/app/views/shared/_google_maps_js.html.haml) as `GOOGLE_MAPS_API_KEY` so I was thinking that we should put close to each other.

Good point, I agree with that reasoning.

Reply:
Created an [issue](https://github.com/openfoodfoundation/openfoodnetwork/issues/11783) and PR for .present?
Added `{% if google_maps_region is defined %}`